packages/product-react/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,152 @@
+import clsx from 'clsx';
+import React from 'react';
+import IconWrapper from '../Common/IconWrapper';
+import RTLWrapper from '../Common/RTLWrapper';
+import FormItemWrapper from '../Common/FormItemWrapper';
+import FieldWrapper from '../Common/FieldWrapper';
+
+import './Select.scss';
+
+export enum IconPosition {
+  Prepend = 'PREPEND',
+  End = 'END',
+  Start = 'START'
+}
+
+export interface ISelectProps extends React.HTMLProps<HTMLSelectElement> {
+  active?: boolean;
+  compact?: boolean;
+  error?: boolean;
+  disabled?: boolean;
+  rtl?: boolean;
+  formItem?: boolean;
+  className?: string;
+  icon?: React.ReactNode;
+  value?: number | string;
+  iconPosition?: IconPosition;
+  options: Option[];
+}
+
+interface Option {
+  value: number | string;
+  label: string;
+  disabled?: boolean;
+}
+
+export const Select: React.FC<ISelectProps> = ({
+  id,
+  active,
+  label,
+  compact,
+  disabled,
+  error,
+  placeholder,
+  className,
+  options,
+  rtl,
+  formItem,
+  icon,
+  iconPosition,
+  onChange,
+  onFocus,
+  onBlur,
+  value,
+  ...rest
+}) => {
+  let optionItems: React.ReactNode[] = [];
+  const [currValue, setValue] = React.useState(value);
+  const [activeClass, setActiveState] = React.useState(active);
+
+  React.useEffect(() => {
```

Comment:
Is this effect necessary? It looks like you are already initing the value on line 57.

Reply:
yes you are right , forgot to remove it

Reply:
done